### PR TITLE
Fixes.

### PR DESCRIPTION
**Pull request by Lukc**
Fixes the following issues:

- Elves used to be unable to recruit through research.
- Diplomacy with a faction stopped once a recruit from that faction had been obtained.
- Elves could start with human leaders.

### CODE REVIEW

File: utils/ANLEra_leader_options.cfg [hunk 1]
```diff
@@ -487,10 +492,12 @@ Help an ally with their research"
...
-                            {AND_NOT_RECRUIT "Dwarvish Fighter"}
-                            {AND_NOT_RECRUIT "Dwarvish Guardsman"}
-                            {AND_NOT_RECRUIT "Dwarvish Thunderer"}
-                            {AND_NOT_RECRUIT "Dwarvish Ulfserker"}
+                            [and]
+                                {NOT_RECRUIT "Dwarvish Fighter"}
+                                {OR_NOT_RECRUIT "Dwarvish Guardsman"}
+                                {OR_NOT_RECRUIT "Dwarvish Thunderer"}
+                                {OR_NOT_RECRUIT "Dwarvish Ulfserker"}
+                            [/and]
```
Comment (sevu) [inline utils/ANLEra_leader_options.cfg:500]: You don't need the [and] tags, and is the default.

Comment (Lukc) [reply utils/ANLEra_leader_options.cfg:500]: Right. Maybe this wasn’t the right way to do this, as I mostly wanted to group the `[or]`s together.

Comment (sevu) [reply utils/ANLEra_leader_options.cfg:500]: Pardon, they are needed.

**Comment by sevu** [comment]
I'm a bit late here … but this should be merged soon™, as it contains a bugfix @Robertdebrus 

Btw., I'm going to be in Strasbourg these days @Lukc

**Pull request merged**